### PR TITLE
Clean up: quantitative value range

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/miscellaneous/quantitativeValueRange.schema.tpl.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "properties": {
     "maxValue": {
       "type": "number",
-      "_instruction": "Add the maximum value measured for this range."
+      "_instruction": "Enter the maximum value for this quantitative value range."
```
Comment (lzehl) [inline schemas/miscellaneous/quantitativeValueRange.schema.tpl.json]: I think we should not use "quantitative value" here but keep the old phrasing

File: schemas/miscellaneous/quantitativeValueRange.schema.tpl.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     },
     "minValue": {
       "type": "number",
-      "_instruction": "Add the minimum value measured for this range."
+      "_instruction": "Enter the minimum value for this quantitative value range."
```
Comment (lzehl) [inline schemas/miscellaneous/quantitativeValueRange.schema.tpl.json]: see above

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by lzehl** [comment]
@apdavison I need your input here

**Comment by apdavison** [comment]
I don't feel strongly about this. "Enter the maximum value." and "Enter the minimum value." would be sufficiently clear in context.

**Review by lzehl — APPROVED** [review]
changed the instructions to @apdavison suggestion.

**Comment by lzehl** [comment]
@Peyman-N or @apdavison or @aeidi89 feel free to merge if you also approve now

**Review by aeidi89 — APPROVED** [review]
Looks good.

**Pull request merged**